lighthouse-core/report/v2/renderer/details-renderer.js

```diff
@@ -81,6 +83,36 @@ class DetailsRenderer {
     element.appendChild(items);
     return element;
   }
+
+  /**
+   * @param {!DetailsJSON} list
```

Comment:
this isn't a `DetailsJSON`, though (specifically `list.items` isn't a `Array<DetailsJSON>|undefined`). Maybe you want to create a new `CardsDetailsJSON` or whatever? I'm also not clear where this leaves us on the whole "new `details` property has a standard structure" thing